platform/8.0/_index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Jakarta Platform Spec 8.0"
+date: 2019-07-18T16:10:38-04:00 (use "date +%FT%T%z")
+summary: "Initial release of the Jakarta EE 8 Platform"
+---
+The Jakarta EE Platform defines a standard platform for hosting Jakarta EE applications.
+
+* [Jakarta EE Platform 8.0 Specification Document](./Platform-spec-8.0.pdf) (PDF)
```

Comment:
We never discussed this explicitly but should we use "8.0" or just "8" for the version number?
For Java EE, we decided we would never do "dot" releases.

Reply:
I kind of like the dot release, even for the Platform.  But, I'm flexible either way.  Maybe we should discuss at the Spec Committee?

Reply:
Yes, we should discuss, later...